docs/source/datasets.rst

```diff
@@ -81,6 +81,13 @@ SPEECHCOMMANDS
   :special-members:
 
 
+TEDLIUM
+~~~~~~~~~~~~~~
+
+.. autoclass:: TEDLIUM
+  :members: __getitem__
+  :special-members: get_phoneme_dict
```

Comment:
@mthrok What does special-members mean here? I interpretate it as extra functions to include in the docs? Thats why I included get_phoneme_dict

Reply:
These are Sphinx's directive. Check out [their documentation](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html)

 - `:members:` is where you list the members you want document
 - `:special-members:` is where you list the special methods like `__init__`, `__len__`, `__getitem__` etc...

I think the other documentations are wrong, (`__getitem__` should be under `:special-member:` but it will not show up either way because they don't have a docstring.)

I think you can just do `.. autoclass:: TEDLIUM` and the rest (`get_phoneme_dict`) will be handled.

You can build the documentation and check how the resulting documentation looks like.

```
cd docs
pip install -r requirements.txt
make html
# open ./build/html/index.html
```